html2docx.py

```diff
@@ -3,13 +3,40 @@
 from io import BytesIO
 from typing import List, Optional, Tuple
 
+from cssutils import parseStyle
+from cssutils.css import CSSStyleDeclaration
 from docx import Document
 from docx.text.paragraph import Paragraph
 from docx.text.run import Run
 
 WHITESPACE_RE = re.compile(r"\s+")
 
 
+def attr_to_css(attr: Tuple[str, Optional[str]]) -> Optional[CSSStyleDeclaration]:
+    if attr[0] == "style":
+        return parseStyle(attr[1])
+    return None
+
+
+def get_attrs(attrs: List[Tuple[str, Optional[str]]]) -> List[str]:
+    """Return Font style names based on tag style attributes
+
+    :param attrs:
+        a list of attribute name plus attribute value tuples.
+    :returns:
+        a list of style names to be applied to Run Font property
+    """
+    args = []
+    for attr in attrs:
+        css = attr_to_css(attr)
+        if css:
```

Comment:
Removing this guard does not cause any test failures. When is `css` `None`? A test should be added.